### PR TITLE
EIP-1: EIP Purpose and Guidelines

### PR DESCRIPTION
**Pull request by contextlimit**

### CODE REVIEW

File: EIPS/EIP-1.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+We intend EIPs to be the primary mechanisms for proposing new features, for collecting community input on an issue, and for documenting the design decisions that have gone into the ecosystem. Because the EIPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+For implementers, EIPs are a convenient way to track the progress of their implementation. Ideally each implementation maintainer would list the EIPs that they have implemented. This will give end users a convenient way to know the current status of a given implementation or library.
+
+### EIP Types
+
+There are three types of EIP:
+
+-   A **Standard Track EIP** describes any change that affects most or all implementations, such as a change to the the network protocol, proposed application standards/conventions, or any change or addition that affects the interoperability of applications. Furthermore Standard EIPs can be broken down into the following categories.
```
Comment (WalidOfNow) [inline EIPS/EIP-1.md]: "change to the the network"

File: EIPS/EIP-1.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+If you are interested in assuming ownersEIP of an EIP, send a message asking to take over, addressed to both the original author and the EIP editor. If the original author doesn't respond to email in a timely manner, the EIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
+
+### EIP Editors
+
+The current EIP editors are
+
+` * Eric Muyser (@ericmuyser)`
+
+` * Walid Habboul (@walid_wah)`
```
Comment (WalidOfNow) [inline EIPS/EIP-1.md]: Walid Al Habboul

File: EIPS/EIP-1.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    -   **Application** - application-level standards and conventions, including contract standards such as token standards, URI schemes, library/package formats, and wallet formats.
+
+-   An **Informational EIP** describes a design issue, or provides general guidelines or information to the community, but does not propose a new feature. Informational EIPs do not necessarily represent community consensus or a recommendation, so users and implementers are free to ignore Informational EIPs or follow their advice.
+-   A **Meta EIP** describes a process or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the protocol itself. They may propose an implementation, but not to the codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in development. Any meta-EIP is also considered a Process EIP.
```
Comment (WalidOfNow) [inline EIPS/EIP-1.md]: probably replace "like" with "similar to"?

**Review by WalidOfNow — APPROVED** [review]

**Comment by contextlimit** [comment]
Thanks @0xwalid 👊 💥 👊

**Pull request merged**